.github/workflows/sanitycheck_tests.yml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Zephyr Sanitycheck TestSuite
+
+on:
+  push:
+    paths:
+    - 'scripts/sanity_chk/sanitylib.py'
```

Comment:
what about other files under scripts/sanity_chk?

Reply:
Sorry, I am working on figuring this out. I should have converted this PR to draft before submitting the changes.

Reply:
Fixed this. Its running now. Please review this now. Thanks.

Reply:
Any reason this isn't just `scripts/sanity_chk/**`?

Reply:
Yes, updated this to trigger on scripts/sanity_chk/**. thanks